src/NuGet.Core/NuGet.Packaging/Signing/Package/SignedPackageArchive.cs

```diff
@@ -77,19 +68,11 @@ public SignedPackageArchive(Stream packageReadStream, Stream packageWriteStream)
                 throw new SignatureException(Strings.SignedPackageNotSignedOnRemove);
             }
 
-            using (var writeZip = new ZipArchive(_zipWriteStream, ZipArchiveMode.Update, leaveOpen: true))
-            {
-                writeZip.GetEntry(_signingSpecification.SignaturePath).Delete();
-            }
+            Zip.GetEntry(_signingSpecification.SignaturePath).Delete();
         }
 
         protected override void Dispose(bool disposing)
```

Comment:
you should just remove this override if it isn't needed

Reply:
Override is needed because the parent class has a method with the same signature.